react/features/base/lib-jitsi-meet/functions.any.js

```diff
@@ -45,6 +45,28 @@ export function isAnalyticsEnabled(stateful: Function | Object) {
     return !(disableThirdPartyRequests || analytics.disabled);
 }
 
+/**
+ * Determines whether an avatar URL is valid.
+ *
+ * @param {Function|Object} stateful - The redux store, state, or
+ * {@code getState} function.
+ * @param {string} avatarURL - The avatar url to check.
+ * @returns {boolean} If the URL matches the origin and disableThirdPartyRequests
+ * is defined, or the URL matches the gravatarBaseURL, {@code true};
+ * {@code false}, otherwise.
+ */
+export function isAvatarUrlValid(stateful: Function | Object, avatarURL: string) {
+    const { disableThirdPartyRequests, gravatarBaseURL } = toState(stateful)['features/base/config'];
+
+    if (disableThirdPartyRequests) {
+        return avatarURL && avatarURL.startsWith(window.location.origin);
```

Comment:
This won't work on mobile because there is no window.location. You can get the URL from features/base/connection

Reply:
I have updated this part and now use locationURL from features/base/connection as you suggested